docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,227 @@
+change should appear.
+1. Next click the user name you want to enable and disable automatic login.
+1. Then on the top right side of the window click `Unlock...` button.
+1. You will be required to enter your user password to continue.
+1. You should now be able to toggle Automatic Login button to enable.
+1. Restart your system for changes to take effect.
+
+## Download and install Checkbox
+
+1. Open the terminal and type in the below-described commands to install
+Checkbox and all its dependencies:
+
+    ```bash
+    sudo add-apt-repository ppa:hardware-certification/public
+    sudo apt-get update
+    sudo apt-get install checkbox-ng plainbox-provider-resource-generic \
+    plainbox-provider-certification-client plainbox-provider-checkbox \
+    canonical-certification-client
+    ```
+
+## Run automatic validation
+
+1. Start checkbox. To do this, open the terminal and run the following command:
+
+    ```bash
+    sudo checkbox-cli
+    ```
+
+    After using the above-mentioned command, the following menu should appear:
+
+    ```bash
+    ┌──────────────────────────────────────────────────────────────────────────────┐
+    │    (X) (Deprecated) Fully Automatic Client Certification Tests               │
+    │    ( ) After suspend Dock Cert blocker tests                                 │
+    │    ( ) After suspend Dock Cert tests                                         │
+    │    ( ) After suspend Dock Hot-plug Cert tests                                │
+    │    ( ) After suspend Dock Hot-plug tests                                     │
+    │    ( ) After suspend LED and oops tests (using special sleep key)            │
+    │    ( ) After suspend LED and oops tests (using special sleep key, cert.      │
+    │        blockers only)                                                        │
+    │    ( ) After suspend automated USB 3 write/read/compare tests on storage     │
+    │        devices                                                               │
+    │    ( ) After suspend automated USB write/read/compare tests on storage       │
+    │        devices                                                               │
+    │    ( ) After suspend reference tests                                         │
+    │    ( ) After suspend reference tests (automated)                             │
+    │    ( ) After suspend reference tests (certification blockers only)           │
+    │    ( ) After suspend tests (discrete GPU automated)                          │
+    │    ( ) After suspend tests (discrete GPU manual)                             │
+    │    ( ) After suspend tests (discrete GPU)                                    │
+    │    ( ) After suspend tests (discrete GPU, certification blockers only)       |
+            .
+            .
+            . etc.
+    └──────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+> Marking the first option starts the full automate check.
```

Comment:
Improper formatting. This type of formatting will delete the numbering

Reply:
repaired